sistr/src/serovar_prediction/__init__.py

```diff
@@ -161,11 +168,17 @@ def search_for_wzy(self):
                 self.wzy_prediction.serogroup = None
                 return
 
-            if top_result_length < 600:
+            if top_result_length < 300:
```

Comment:
Same comment here about `top_result_length`.

Reply:
Thanks for catching that, it has been fixed in commit 3ef9915..3925340